website/snippets/_dbt-package-hub-env-var.md

```diff
@@ -0,0 +1,22 @@
+If you're hosting an internal version of the [dbt Labs hosted package hub](https://hub.getdbt.com/), you can set the [`DBT_PACKAGE_HUB_URL`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/clients/registry.py#L23) environment variable to point to your instance of the dbt package hub. This allows you to install packages from your self-hosted version. 
+
+It provides organizations with the flexibility, control, and security when managing dbt packages:
+
+- Host internal dbt packages for organizational use &mdash; Useful for sharing sensitive or proprietary data models, macros, or other items across the organization.
+- Manage custom versions of public dbt packages &mdash; Useful for maintaining a fork of a public package with customizations or bug fixes.
+- Operate in secure, regulated, or air-gapped environments &mdash; Useful for organizations that require strict control over the software they use.
+- Gain control over package updates and dependencies &mdash; Useful for organizations that need to manage dependencies and updates carefully.
```

Comment:
```suggestion
- Gain control over package updates and dependencies &mdash; Handy for organizations that need to manage dependencies and updates carefully.
```

Reply:
Hiya @mirnawong1 

These three are non-blockers but I thought it might be worthwhile using other words instead of repetition?

Kind Regards
Natalie